.travis.yml

```diff
@@ -9,8 +14,16 @@ branches:
     - /^\d+\.\d+(\.\d+)?(-\S*)?$/
     #hotfix branches (not being too strict on purpose)
     - /^hf-\.*$/
+    
+before_install:
+  - docker pull nmldiegues/rserve-caret-java:latest
+  - docker run -v $(pwd):/opt/openml -d --name test-container nmldiegues/rserve-caret-java:latest
+
+install:
+  - docker exec -it test-container mvn install -f /opt/openml/pom.xml -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
Why do we need this besides the "script" below?

Reply:
I though that way at least we only download maven dependencies once 😛 (will download again for deployments, but those are less frequent).